desktop/apps/artwork/client/index.coffee

```diff
@@ -161,5 +172,8 @@ module.exports =
       req: user: CurrentUser.orNull()
     }
       .then (data) ->
-        renderTemplates(data)
-        exec init
+        renderTemplates(extend data, {
+          auctionContextId: AUCTION.id
```

Comment:
I was wracking my brain on how this was missed and figured it out -- I had initially grabbed this value from a query param which would safely fallback to undefined only to later swap it out once @sweir27 brought the bootstrapping mechanism to my attention; swapped it out for that but didn't follow up with previous tests. Damnit! thanks again for the fix.